### ISSUE TITLE
Peak element In a Mountain Array Using Binary Search

### ISSUE THREAD

**Issue by GundaSudarrshan** [HackClubRAIT/Wizard-Of-Docs#117]
### Description 
This Problem solves one of the Interesting Question , That is to Find out the highest element in an array that contains both Ascending and Descending Elements, so we need to find that element from which the list of elements changes from 
Ascending to Descending (or) Descending to Ascending.

### Programming language
- [ ] C
- [ ] C++
- [x] Java
- [ ] Python

#### Are you contributing under any open-source program ?
Hack ClubRait Summer of Code ( HSOC -22)

<hr/>

**Comment by GundaSudarrshan** [comment]
@Chaitalishetty  can You look up and this to this Issue, add labels and Assign me this Issue.

**Comment by Chaitalishetty** [comment]
@GundaSudarrshan Assigning this issue to you. Happy Coding !

**Comment by GundaSudarrshan** [comment]
@Chaitalishetty  I Wanted to Create a Document about Binary Search and solve that Problem in Two Languages(Python, Java) 
Should I make Another Issue for Binary Search Document 
and another doubt is if I do it in Two Languages Will this be considered as single-issue or Multiple Issues.

**Comment by Chaitalishetty** [comment]
> @Chaitalishetty I Wanted to Create a Document about Binary Search and solve that Problem in Two Languages(Python, Java) Should I make Another Issue for Binary Search Document and another doubt is if I do it in Two Languages Will this be considered as single-issue or Multiple Issues.

Yeah you can create a document and add it in the same issue also if you're doing it in multiple languages, it will be considered as single issue

**Comment by GundaSudarrshan** [comment]
ok I will update sooner in Python also
